mscore/abstractdialog.cpp

```diff
@@ -28,7 +28,6 @@ AbstractDialog::AbstractDialog(QWidget * parent, Qt::WindowFlags f)
 
 AbstractDialog::~AbstractDialog()
       {
-
```

Comment:
More a general question: is an empty destructor needed at all?

Reply:
I would not know why... so tried removing it. Seems that doesn't cause problems.